docs/getting-started/01-installation.md

```diff
@@ -0,0 +1,36 @@
+# Installation
+
+For many, the easiest way to install Prism is as a node module. 
+
+```bash
+npm install -g @stoplight/prism-cli
+# or
+yarn global add @stoplight/prism-cli
+```
+
+## Executable Binaries
+
+For users without Node and/or NPM/Yarn, we provide standalone packages for [all major platforms](https://github.com/stoplightio/prism/releases). The quickest way to install the appropriate package for your operating system is via this shell script:
+
+```bash
+curl -L https://raw.githack.com/stoplightio/prism/master/install | sh
+```
+
+<!-- theme: info -->
+> The binaries do _not_ auto-update, so you will need to run it again to install new versions.
+
+## Docker
+
+Prism is available as a Docker image. We recommend specifying the major version you'd like to use as a tag:
+
+```bash
+docker run -P stoplight/prism:3 mock -h 0.0.0.0 api.oas2.yml
+```
+
+If the document you want to lint is on your computer, you'll need to mount the directory where the file resides as a volume:
+
+```bash
+docker run --rm -it -v $(pwd):/tmp stoplight/spectral lint "/tmp/file.yaml"
```

Comment:
```suggestion
docker run --init --rm -it -v $(pwd):/tmp -P stoplight/prism:3 mock -h 0.0.0.0 "/tmp/file.yaml"
```

Reply:
I'm glad you're noticing all these issues because this documentation has been in master for ages. I'm just moving existing content around. 😬

Reply:
I'm afraid it's incorrect — the current documentation is fine — I guess this is a leftover from Spectral's copy paste:

https://github.com/stoplightio/prism/blob/master/docs/getting-started/installation.md#docker